askama_tide/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
The license files should be symlinks, as in the other integration crates.

Reply:
resolved in 413d4d4ce9fd197c69f896fe6d1eefea6ff96c01